### PR TITLE
Expose a bunch of additional parameters on SceneBuilder

### PR DESCRIPTION
**Pull request by hrydgard**
Pretty straightforward.

### CODE REVIEW

File: physx/src/scene.rs [hunk 1]
```diff
@@ -563,6 +568,11 @@ impl Default for SceneBuilder {
             use_ccd: false,
             enable_ccd_resweep: false,
             solver_type: SolverType::PGS,
+            enhanced_determinism: false,
+            friction_every_iteration: false,
+            stabilization: false,
+            adaptive_force: false,
+            ccd_max_passes: 1,
```
Comment (tgolsson) [inline physx/src/scene.rs:575]: Are these the default settings for PhysX?

**Review by emilk — APPROVED** [review]
🥳

**Review by tgolsson — APPROVED** [review]
LGTM with the caveat below.

File: physx/src/scene.rs (hunk 1 above)
Comment (hrydgard) [reply physx/src/scene.rs:575]: Yes.

Comment (tgolsson) [reply physx/src/scene.rs:575]: Perfect!

**Pull request merged**